### ISSUE TITLE
Notification for releasable charters

### ISSUE THREAD

**Issue by StephanMa** [icaruseu/mom-ca#958]
Archivists should get a notification, too if a charter is suggested for publication.

To Do:

- Send mail to archivist
- add dummy archivist to publication.app.xml

**Comment by GVogeler** [comment]
first point is the same as #622

**Comment by GVogeler** [comment]
(general goal: replace moderator with archivists or collection owner)

**Comment by StephanMa** [comment]
I wrote with @yngwi about this topic.
Remove moderators totally will take a lot of affort and cant be done "fast".

**Comment by yngwi** [comment]
Yes, to move forward I suggest we don't remove moderators, we just add an additional (default) path to getting charter edits released. This way a university-lecture (for instance) in Naples with Antonella Ambrosio as Moderator would still be possible. In the end it's of course not ideal but in this specific case I think adding a default "archivist" not necessarily hard-coded into the archive-xml data but configurable would be a big step forward.

**Comment by GVogeler** [comment]
Yes, that's true! I even considered to set a default moderator for each new user and add functionality to let the user change the moderator. But that would certainly be a separate issue.

**Comment by GVogeler** [comment]
see PR #1037 and #1035
